front/app/components/admin/ContentBuilder/Widgets/Iframe/utils.test.ts

```diff
@@ -132,20 +132,12 @@ const validPlatformLinks = [
 ];
 
 describe('isValidUrl', () => {
-  // I'm using this structure to make it easy to identify which platform has a failing test
   it.each(validPlatformLinks)(
     'should return [true, "whitelist"] for valid url %s',
```

Comment:
Minor: we can edit this test for only one valid url. The old array was meant to limit to only the ones we wanted but now that we have opened it up, we can add one test for a valid url and another for an invalid one.